pkg/kubelet/kuberuntime/util/util_test.go

```diff
@@ -0,0 +1,227 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	v1 "k8s.io/api/core/v1"
+	runtimeapi "k8s.io/cri-api/pkg/apis/runtime/v1"
+	kubecontainer "k8s.io/kubernetes/pkg/kubelet/container"
+)
+
+func TestPodSandboxChanged(t *testing.T) {
+	for desc, test := range map[string]struct {
+		input             *v1.Pod
+		status            *kubecontainer.PodStatus
+		expectedChanged   bool
+		expectedAttempt   uint32
+		expectedSandboxID string
+	}{
+		"Pod with no existing sandboxes": {
```

Comment:
Could you include keys in these structs? - it's hard to skim these when reading/modifying these in the future when relying on inference.

Reply:
fixed